CAFAna/Core/Hist.cxx

```diff
@@ -154,6 +154,16 @@ namespace ana
     return ret;
   }
 
+  //----------------------------------------------------------------------
+  Hist Hist::AdoptErrors(Eigen::ArrayXd&& v, Eigen::ArrayXd&& e)
```

Comment:
I was confused by this name for a moment because somehow I thought it would only set the errors. Maybe `AdoptWithErrors`?